Assets/Scripts/GameManagement/GameStates/InventoryState.cs

```diff
@@ -0,0 +1,29 @@
+using GameStateManagement;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace GameStateManagement
+{
+    public class InventoryState : IGameState
+    {
+        private InventoryController invController;
+
+        public InventoryState(InventoryController iControl)
+        {
+            invController = iControl;
+        }
+
+        public void EnterState()
+        {
+            invController.GetInventoryPanel().SetActive(true);
+            Time.timeScale = 0f;
+        }
+
+        public void ExitState()
+        {
+            invController.GetInventoryPanel().SetActive(false);
+            Time.timeScale = 1f;
```

Comment:
do you need to set the time scale here? you said you had a pause system things inherited from? is that being used at all?

Reply:
Whenever we implemented the game state manager, that included a Pause State that was being used for the pause menu - I missed that when I was making a pause system so I decided to go with the game state manager instead since it was a nicer system. Mine is not being used at all right now.

Reply:
Do you think you're gonna integrate it all before the merge from this pr or save it for later?